src/components/CreateVmWizard/steps/Storage.js

```diff
@@ -241,24 +240,18 @@ class Storage extends React.Component {
         editView: (value, { rowData }) => {
           const row = this.state.editing[rowData.id]
           const sizeGiB = row.size / (1024 ** 3)
-          const { invalidSizeValue } = row
 
           return <div className={style['disk-size-edit']}>
-            <FormGroup
-              validationState={invalidSizeValue && 'error'}
-              className={style['form-group-edit']}
-            >
+            <div>
```

Comment:
Could you please make it clear why you removed `FormGroup` component? Or why do we prefer `div` instead of `FormGroup`? Usually, `FormControl` is used inside of `FormGroup`. It seems to be working and being displayed okay as it is. I am just asking :)

Reply:
`FormControl` itself is spreading over the whole cell and hides every other "sibling" component so it have to be wrapped by another component, It could be `FormGroup` but I thought `div` is better because it's simpler and enough for this job...